Source/Private/ASDisplayNode+AsyncDisplay.mm

```diff
@@ -332,7 +332,7 @@ - (void)displayAsyncLayer:(_ASDisplayLayer *)asyncLayer asynchronously:(BOOL)asy
     return;
   }
   
-  ASDisplayNodeAssert(_layer, @"Expect _layer to be not nil");
+  ASDisplayNodeAssert(layer, @"Expect _layer to be not nil");
```

Comment:
The instance lock is no longer held by the time this line is reached. It's safer this way.